src/Tests/PathautoLocaleTest.php

```diff
@@ -27,14 +31,19 @@ class PathautoLocaleTest extends WebTestBase {
    *
    * @var array
    */
-  public static $modules = array('node', 'pathauto', 'locale');
+  public static $modules = array('node', 'pathauto', 'locale', 'content_translation');
 
   /**
-   * Admin user.
-   *
-   * @var \Drupal\user\UserInterface
+   * {@inheritdoc}
    */
-  protected $adminUser;
+  protected function setUp() {
+    parent::setUp();
+
+    // Create Article node types.
+    if ($this->profile != 'standard') {
```

Comment:
You possibly copied this check from somewhere but it doesn't make sense. there are no standard profile tests, it's always testing.